src/pages/Index.vue

```diff
@@ -5,24 +5,28 @@
         <h1 class="uppercase">{{ $t('supportTogether') }}</h1>
         <div class="row justify-between">
           <router-link
-            class="col-xs-12 col-md-4 q-mt-xs uppercase"
-            to="/get-help"
+            class="col-xs-12 col-md-5 q-mt-xs uppercase my-card"
+            to="/help"
           >
             <MainButton content="want-help" />
           </router-link>
-          <router-link class="col-xs-12 col-md-4 q-mt-xs uppercase" to="/help">
+          <router-link class="col-xs-12 col-md-5 q-mt-xs uppercase my-card" to="/get-help">
             <MainButton content="need-help" />
           </router-link>
         </div>
       </div>
+      <div class="mouse">
```

Comment:
What's the purpose of these empty containers ?

Reply:
https://xd.adobe.com/view/ecf2b23c-6d61-42a4-5842-09283c46e28c-9ac3/
its a icon with animation, to offer the user to scroll, see the design template